README.md

```diff
@@ -1,4 +1,5 @@
-# XGI
+### XGI ###
```

Comment:
what's the difference with the previous format?

Reply:
The difference is that using the single hashtag is a title, whereas the triple hashtags on either side format it the same but exclude it from the table of contents.